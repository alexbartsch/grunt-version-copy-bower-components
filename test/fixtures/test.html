<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>Test Page</title>

  <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.css">
  <link rel="stylesheet" href="css/test.css">
</head>
<body>
  <script src="bower_components/jquery/dist/jquery.js"></script>
  <script src="bower_components/handlebars/handlebars.js"></script>
  <script src="bower_components/ember/ember.js"></script>
  <script src="bower_components/ember-model/ember-model.js"></script>
  <script src="js/app.js"></script>
  <script src="bower_components/bootstrap/dist/js/bootstrap.js">
</body>
</html>

