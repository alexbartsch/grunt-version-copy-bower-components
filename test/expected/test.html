<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>Test Page</title>

  <link rel="stylesheet" href="libs/bootstrap-3.1.1/dist/css/bootstrap.min.css">
  <link rel="stylesheet" href="test.css">
</head>
<body>
  <script src="libs/jquery-2.1.0/dist/jquery.min.js"></script>
  <script src="libs/handlebars-1.3.0/handlebars.min.js"></script>
  <script src="libs/ember-1.5.0/ember.min.js"></script>
  <script src="js/app.js"></script>
  <script src="libs/bootstrap-3.1.1/dist/js/bootstrap.min.js">
</body>
</html>





